apps/log/lib/log/model/log.ex

```diff
@@ -61,7 +61,7 @@ defmodule Helix.Log.Model.Log do
     %__MODULE__{}
     |> cast(params, @creation_fields)
     |> validate_required(@required_fields)
-    |> put_primary_key()
+    |> put_change(:log_id, PK.pk_for(__MODULE__))
```

Comment:
> Because when you assoc something to a changeset (and the schema defines that association), ecto will know what to do on insert (ie: it'll use the returned id and in the darkness bind them)

Damn Ecto and its black magic making our life easier again :stuck_out_tongue: